source/snippets/listAllOrgSheets.md.erb

```diff
@@ -49,7 +49,7 @@ sheets = response.data
 
 `GET /users/sheets`
 
-Gets the list of all Sheets owned by the members of the account (organization). 
+Gets a summarized list of all Sheets owned by the members of the account (organization). 
```

Comment:
I'm afraid this will be interpreted as returning a partial list. We are returning a full list - with summary details for each sheet. Same for listSentUpdateRequests.md.erb & listUpdateRequests.md.erb

Reply:
Good point. I thought I had figured out an elegant solution, but I see that it is ambiguous. I rewrote 27 topics using "summarized" so I can fix all of those instances. How about this as a generic wording fix that I may have to tweak a bit in some of those cases? "Gets a list of all Sheets... " And then a 2nd sentence saying: "The list contains only summary details for each Sheet."

Reply:
It looks like you only get an abbreviated object for Sheets and Workspaces, so I've corrected the doc to reflect that.